content/applications/sales/sales/amazon_connector/features.rst

```diff
@@ -60,6 +60,16 @@ The following table lists capabilities provided by Odoo when using the Amazon Co
 |                           |                            | status synchronized from Odoo.      |
 +---------------------------+----------------------------+-------------------------------------+
 
+.. important::
+   The stock synchronization does **not** currently support selling the same product as :abbr:`FBM
+   (Fulfilled By Merchant)` *and* :abbr:`FBA (Fulfilled By Amazon)`.
+
+   At times, when stock is sent for all products, it triggers a stock problem with Amazon, where
+   Amazon incorrectly thinks the FBA product has some quantity in FBM.
+
+   As a result, Amazon then sells it as FBM, instead of taking from their own warehouse. Odoo
+   developers are currently working on resolving this issue to avoid future discrepancies.
```

Comment:
Please add abbr throughout :)
```suggestion
   The stock synchronization does **not** currently support selling the same product as :abbr:`FBM
   (Fulfilled By Merchant)` *and* :abbr:`FBA (Fulfilled By Amazon)`.

   At times, when stock is sent for all products, it triggers a stock problem with Amazon, where
   Amazon incorrectly thinks the :abbr:`FBM (Fulfilled By Merchant)` product has some quantity in
   :abbr:`FBM (Fulfilled By Merchant)`.

   As a result, Amazon then sells it as :abbr:`FBM (Fulfilled By Merchant)`, instead of taking from
   their own warehouse. Odoo developers are currently working on resolving this issue to avoid
   future discrepancies.
```